hal/src/tracker/hal_platform_config.h

```diff
@@ -48,3 +48,9 @@
 #define HAL_PLATFORM_MAX_CLOUD_CONNECT_TIME (9*60*1000)
 
 #define PRODUCT_SERIES "boron"
+
+#define PLATFORM_TRACKER_MODEL_BARE_SOM_DEFAULT        (0xffff)
```

Comment:
These macros are application specific and likely redundant

Reply:
As we discussed yesterday, I'm adding these here "just in case" for non-tracker-edge use-cases. Doesn't cost us anything, they don't overlap with previous edge-specific defines and are properly namespaced (`PLATFORM_`). No strong opinions on this though.

Reply:
They can stay.  Would you mind adding another macro like this or similiar?
```cpp
#define PLATFORM_TRACKER_MODEL_LAST    (PLATFORM_TRACKER_MODEL_MONITORONE)
```

Reply:
Sure, we can add it.